logger/log-setup.md

```diff
@@ -0,0 +1,16 @@
+# Installation of Logbook
```

Comment:
I think I would rename this to README.md and make it a broader document. Logbook Setup could be a section, but we could also include other sections for usage, to-do's, etc.